artifactory.py

```diff
@@ -1268,23 +1296,11 @@ def download_folder_archive(self, archive_type="zip", check_sum=False):
         :param: check_sum (bool): defines of check sum is required along with download
         :return: raw object for download
         """
-        if archive_type not in ["zip", "tar", "tar.gz", "tgz"]:
-            raise NotImplementedError(archive_type + " is not support by current API")
-
-        archive_url = (
-            self.drive
-            + "/api/archive/download/"
-            + self.repo
-            + self.path_in_repo
-            + "?archiveType="
-            + archive_type
+        print(
+            "This method will deprecated in future versions. Consider using path.archive(). "
```

Comment:
Why will it be deprecated?

Reply:
we do not need to methods for the same thing. To make archives working with latest chunk download I had to reconsider archives.
Now, people can use path.archive() instead of _download_folder_archive_. path.archive() does not download explicitelly but provides consistent way with general file download, eg

```python
with path.archive(archive_type="zip", check_sum=False) as archive:
    with open(r"D:\target.zip", "wb") as out:
        out.write(archive.read())

# download folder archive in chunks
path.archive().writeto(output="my.zip", chunk_size=100 * 1024)
```

Reply:
I'm not seeing any reason why should add this deprecation warning. There's no future plans about adding a new functionality that will break this method , isn't there?

Reply:
If it works exaclty like `archive` does, lets just add an alias it `download_folder_archive = archive`

Reply:
I agree that `archive` is a better name for it

Reply:
but why should we maintain any function, that is repeated
if you will add functionality (or API changes), then we have to update multiple methods. What is the sense to keep two methods that do the same thing?

new way provides better and consistent way of downloading artifacts as archive that is compatible with other functions.

old function supported only stream download

Reply:
> If it works exaclty like `archive` does, lets just add an alias it `download_folder_archive = archive`

Sorry, missed that comment
It provides same functionality in more consistent way, but works differently. It does not return accessor
Your first apply "archive" to path, then you work with path as you usually will do
Eg download or download in chunks

You can compare difference of usages in examples

Reply:
@beliaev-maksim I'd prefer to keep both methods here